packages/tools/docker/cluster/docker-compose.yml

```diff
@@ -47,7 +47,7 @@ services:
         
     cluster0:
       container_name: cluster0
-      image: ipfs/ipfs-cluster:latest
```

Comment:
I assume the latest version introduces some issue or is buggy?
Do we have that context? 
If yes I wonder if we could add a comment and maybe a follow-up ticket to move this back to latest if needed?